rest/taskana-rest-spring/pom.xml

```diff
@@ -164,6 +164,12 @@
       <artifactId>spring-boot-starter-validation</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.tngtech.archunit</groupId>
+      <artifactId>archunit</artifactId>
+      <version>0.20.1</version>
+      <scope>test</scope>
+    </dependency>

```

Comment:
```suggestion
    <dependency>
      <groupId>com.tngtech.archunit</groupId>
      <artifactId>archunit-junit5</artifactId>
      <version>${version.archunit}</version>
      <scope>test</scope>
    </dependency>

```

Reply:
Please replace `archunit-junit5-api` and `archunit-junit5-engine` dependency in taskana-core/pom.xml with this aggregator dependency. Thanks!

Reply:
according to the module guide of archunit, this module does not exist